web_widget_kanban_many2one/static/src/xml/web_widget_kanban_many2one.xml

```diff
@@ -0,0 +1,9 @@
+<template>
+    <t t-name="Kanban_editable_temp">
+        <span class="container">
+            <select class="form-control text-success selectM2MField" name="id">
```

Comment:
name="id"?

Reply:
It a select like in forms so we need to define an id so that when a person selects a value it is captured as id=1,2,3,4 to be used by the widget.